_extras/guide.md

```diff
@@ -27,13 +27,15 @@ for a look at how using checklists can save lives (and make many other things be
 
 > ## Setting Up [Trainer]
 >
+> 1.  Create an Etherpad (http://pad.software-carpentry.org/<nameofevent>
```

Comment:
I have used the doc. I agree that the link should go in place of etherpad link. One issue we had with the doc is, people cannot chat if they're not logged in in google account. This is bigger of an issue in online training, since it depends on chat a lot.